code/modules/crafting/guncrafting.dm

```diff
@@ -15,6 +15,12 @@
 	icon = 'icons/obj/improvised.dmi'
 	icon_state = "riflestock"
 
+/obj/item/weaponcrafting/heatsink
+	name = "heatsinked tube"
+	desc = "A rather think tube with indents on it meant to disperse heat."
```

Comment:
`think` -> `thin`

but its not written very well...

maybe:

`A remarkably thin tube marked with indents designed to disperse heat.` or something similar